src/cli/cli.cpp

```diff
@@ -2540,6 +2930,19 @@ template <> otError Interpreter::Process<Cmd("fake")>(Arg aArgs[])
 {
     otError error = OT_ERROR_INVALID_COMMAND;
 
+    /**
+     * @cli fake (a,an)
+     * @code
+     * fake /a/an fdde:ad00:beef:0:0:ff:fe00:a800 fd00:7d03:7d03:7d03:55f2:bb6a:7a43:a03b 1111222233334444
+     * Done
+     * @endcode
+     * @cparam fake /a/an @ca{dst-ipaddr} @ca{target} @ca{meshLocalIid}
+     * @par
+     * Sends fake Thread messages.
+     * @note Only for certification test.
```

Comment:
Another variation of the certification/test note

Reply:
Hi @abtink could you please help w/some of Jeff's comments? I know wording is important...I don't want to alter the testing/cert meaning for any of the commands...

What is the best/most consistent way to say this? Thanks!

Reply:
My suggestion is that maybe we mention something like:
> Available when `OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE` is enabled.

(we generally use this pattern when an API requires some OT feature to be enabled).

We can also consider adding that the API is for cert or testing, but it may be already clear from dependency on the `REFRENCE_DEVCIE` config. So may be simpler to just not add anything.

Reply:
@abtink & @Vyrastas Thanks for the feedback; I made some updates....